modules/ztp-image-based-upgrade-extra-manifests-guide.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+// * edge_computing/image-based-upgrade/cnf-understanding-image-based-upgrade.adoc
+
+[id="ztp-image-based-upgrade-extra-manifests-guide_{context}"]
+= Extra manifest guidelines
+
+The {lcao} uses extra manifests to restore your target clusters after rebooting with the new default stateroot deployment and before restoring application artifacts.
+
+Different deployment methods require a different way to apply the extra manifests:
+
+{ztp}:: You use the `target-ocp-version: <target_ocp_version>` label to mark the extra manifests that the {lcao} must extract and apply after the pivot.
```

Comment:
The description for `lca.openshift.io/target-ocp-version-manifest-count` is missing for gitops way, see https://github.com/openshift-kni/lifecycle-agent/blob/main/docs/image-based-upgrade.md#extra-manifests